extension/src/project/addFile2Project.ts

```diff
@@ -3,14 +3,14 @@ import { ProjectTreeProvider, isFileAlreadyInProject, hasFileWithSameName } from
 import { AutoLispExt } from '../extension';
 import * as path from 'path';
 
-export async function AddFile2Project() {
+export async function AddFile2Project(fileList?: vscode.Uri[]) {
```

Comment:
the typescript new syntax is nice for null case

Reply:
Yeah, the changes to this file do really show the value of the Typescript update. I didn't have to add any additional lines to recycle the method!